params/homography_mapping.yaml

```diff
@@ -0,0 +1 @@
+homography_mapping: [[0,1]]
```

Comment:
here is defined between which camera the homography should be computed. For 2 normal camera pair it looks like this: `[[0,1],[2,3]]`